db/migrate/20190315000310_backfill_add_filesize_to_runs.rb

```diff
@@ -0,0 +1,8 @@
+class BackfillAddFilesizeToRuns < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def change
+    Run.in_batches.update_all filesize: 0
```

Comment:
Wow I never knew about `in_batches`! That seems a lot more flexible than `find_each`.

Reply:
Strong migrations actually told me about it because it didn't like adding a new column default since that would rewrite the table.